client/account-recovery/forgot-username/forgot-username-form/index.jsx

```diff
@@ -13,79 +14,131 @@ import Button from 'components/button';
+
+		if ( ! requestError ) {
+			return '';
+		}
+
+		if ( requestError.statusCode === 404 ) {
+			return translate( "We weren't able to find a user with that login information. Please try another one." );
+		}
+
+		return translate( "We've encountered some technical issues. Please try again later." );
+	};
+
+	render() {
+		const {
+			translate,
+			userData,
+			isRequesting,
+			requestError,
+		} = this.props;
+
+		const {
+			firstName,
+			lastName,
+			url,
+		} = userData;
+
+		const isPrimaryButtonEnabled = firstName && lastName && url && ! isRequesting;
 
 		return (
-			<div>
+			<Card>
 				<h2 className="forgot-username-form__title">
 					{ translate( 'Forgot your username?' ) }
 				</h2>
-				<p>{ translate( 'Enter your information to find your username' ) }</p>
-				<Card>
-					<FormLabel>
-						{ translate( 'First Name' ) }
-						<FormInput
-							className="forgot-username-form__first-name-input"
-							onChange={ this.firstNameUpdated }
-							value={ firstName }
-							disabled={ isSubmitting } />
-					</FormLabel>
-					<FormLabel>
-						{ translate( 'Last Name' ) }
-						<FormInput
-							className="forgot-username-form__last-name-input"
-							onChange={ this.lastNameUpdated }
-							value={ lastName }
-							disabled={ isSubmitting } />
-					</FormLabel>
-					<FormLabel>
-						{ translate( "Your site's URL" ) }
-						<FormInput
-							className="forgot-username-form__site-url-input"
-							onChange={ this.siteUrlUpdated }
-							value={ siteUrl }
-							disabled={ isSubmitting } />
-					</FormLabel>
-					<Button
-						className="forgot-username-form__submit-button"
-						onClick={ this.submitForm }
-						disabled={ ! isPrimaryButtonEnabled }
-						primary>
-						{ translate( 'Continue' ) }
-					</Button>
-				</Card>
-			</div>
+				<p>{ translate( 'Enter the following information instead.' ) }</p>
+				<FormLabel>
+					{ translate( 'First Name' ) }
+					<FormInput
+						className="forgot-username-form__first-name-input"
+						onChange={ this.firstNameUpdated }
+						value={ firstName || '' }
+						disabled={ isRequesting } />
+				</FormLabel>
+				<FormLabel>
+					{ translate( 'Last Name' ) }
+					<FormInput
+						className="forgot-username-form__last-name-input"
+						onChange={ this.lastNameUpdated }
+						value={ lastName || '' }
+						disabled={ isRequesting } />
+				</FormLabel>
+				<FormLabel>
+					{ translate( "Your site's URL" ) }
+					<FormInput
+						className="forgot-username-form__site-url-input"
+						onChange={ this.siteUrlUpdated }
+						value={ url || '' }
+						disabled={ isRequesting } />
+				</FormLabel>
+				{
+					requestError && (
+					<p className="forgot-username-form__error-message">
+						{ this.getErrorMessage( requestError ) }
+					</p> )
+				}
+				<Button
+					className="forgot-username-form__submit-button"
+					onClick={ this.submitForm }
+					disabled={ ! isPrimaryButtonEnabled }
+					primary>
+					{ translate( 'Continue' ) }
```

Comment:
Oops. After adding, the i18n-bot [complained](https://github.com/Automattic/wp-calypso/pull/11697#discussion_r106117340). I had asked our i18n folks and the conclusion was that I should remove the context in this case to forbid it from being translated for multiple times.